src/eos-crash-metrics.c

```diff
@@ -137,6 +182,122 @@ normalize_path (char *path)
     path[i] = path[i] == '!' ? '/' : path[i];
 }
 
+static inline void
+sanitize_name (gchar *name)
+{
+  gchar *tmp = strchr (name, '/');
+
+  if (tmp != NULL)
+    *tmp = '\0';
+}
+
+static const char *
```

Comment:
Looks like this function returns a new string, so it should be `char *`, not `const char *`